.github/workflows/release.yml

```diff
@@ -1,71 +1,91 @@
-name: Release
+name: release-please
 
 on:
-  workflow_dispatch:
+  push:
+    branches:
+      - master
 
-env:
-  REGISTRY: ghcr.io
-  IMAGE_NAME: ${{ github.repository }}
+permissions:
+  contents: write
+  pull-requests: write
+  packages: write
 
 jobs:
-  release:
+  release-please:
     runs-on: ubuntu-latest
+    outputs:
+      release_created: ${{ steps.release.outputs.release_created }}
+      release_name: ${{ steps.release.outputs.name }}
+
+    steps:
+      - uses: google-github-actions/release-please-action@v3
+        id: release
+        with:
+          release-type: node
+          package-name: nodecg
+
+  publish:
+    runs-on: ubuntu-latest
+    needs: release-please
+    if: ${{ needs.release-please.outputs.release_created }}
 
     steps:
       - uses: actions/checkout@v3
 
       - uses: actions/setup-node@v3
         with:
           node-version: '18'
+          registry-url: 'https://registry.npmjs.org'
+          cache: 'npm'
 
-      - name: Restore node_modules
-        uses: actions/cache@v3
-        with:
-          path: |
-            node_modules
-          key: npm-${{ runner.os }}-18.x-${{ hashFiles('package-lock.json') }}
-
-      - name: Install dependencies
-        run: npm ci
-
-      - name: Build
-        run: npm run build
-
-      - name: Create zip
-        uses: vimtor/action-zip@v1
-        with:
-          # Be sure to keep this in sync with the `files` array in `package.json`
-          files: build/ scripts/ generated-types/ bundles/.empty_directory db/.empty_directory schemas/ AUTHORS package-lock.json package.json
-          recursive: true
-          dest: nodecg.zip
+      - run: npm ci
+      - run: npm run build
 
-      - name: Semantic Release
-        run: npx semantic-release
+      - name: Publish to github release
+        run: |
+          FILE_NAME=$(npm pack)
```

Comment:
`npm pack` creates same file as the npm package but to .tgz (which should be fine on Windows?)